lib/fs.js

```diff
@@ -385,6 +385,13 @@ fs.readFile = function(path, options, callback) {
   if (!nullCheck(path, callback))
     return;
 
+  // doing justice between linux and windows
+  // for `/` char at the end of file path
+  if (isWindows && (path.slice(-1) === '/')) {
+    throw new errors.Error('ERR_INVALID_FILE_URL_PATH',
+                           'file path must not end with / character');
```

Comment:
This string should be reduced into `'must not end with / character'`

The first two words `'file path'` is redundant because these words were already contained in template:
https://github.com/nodejs/node/blob/a82b1b7b30d52cd6a3ea9f129cdc3d83e0370a34/lib/internal/errors.js#L401